### PR TITLE
Fixes Revolution

### PR DESCRIPTION
**Pull request by Cyberboss**
Guilty

### CODE REVIEW

File: code/game/gamemodes/revolution/revolution.dm [hunk 1]
```diff
@@ -114,6 +115,10 @@
 
 
 /datum/game_mode/revolution/process()
+	var/tick = world.time - (SSticker.round_start_time + 10 * 60 * 30)
+	if(tick > 0 && tick % (10 * 60 * 15) == warning_count)
+		++warning_count
```
Comment (Xhuis) [inline code/game/gamemodes/revolution/revolution.dm]: \>not `warning_count++`

Comment (Cyberboss) [inline code/game/gamemodes/revolution/revolution.dm]: ++X is faster in C++ ironically

Comment (p440) [inline code/game/gamemodes/revolution/revolution.dm]: inc first ask questions later

**Comment by DoctoreMad** [comment]
There seems to be a bug with this PR. It encourages admins to adminbus rev rounds less rather than more.
I think this could be a negative experience for people who play the server, honestly.

**Comment by lzimann** [comment]
bad joke

**Pull request closed without merging**